packages/sdk/Dockerfile

```diff
@@ -143,6 +153,7 @@ COPY --from=su-exec /usr/local/src/su-exec /usr/local/bin/
 COPY --from=crane /usr/local/bin/crane /usr/local/bin/
 COPY --from=devnet /usr/local/lib/node_modules/@cartesi/devnet/export/abi/localhost.json /usr/share/cartesi/
 COPY --from=devnet /usr/local/lib/node_modules/@cartesi/devnet/build/anvil_state.json /usr/share/cartesi/
+COPY --from=graphql /usr/local/bin/rollups-graphql /usr/local/bin/cartesi-rollups-graphql
```

Comment:
there is a confusion about the name of the binary here.
It should already be `cartesi-rollups-graphql` in the distribution, but it's currently `rollups-graphql`.

@fabiooshiro will fix it in a future version distribution.

Reply:
@endersonmaia this is fixed in latest 2.3.4